api/auth/BUILD

```diff
@@ -12,3 +12,21 @@ api_proto_library(
         "//api:address",
     ],
 )
+
+api_proto_library(
+    name = "auth",
+    srcs = ["auth.proto"],
+    deps = [
+        "//api/cert",
+    ],
+)
+
+api_go_proto_library(
+    name = "auth",
+    proto = ":auth",
+    deps = [
+        "//api:address_go_proto",
```

Comment:
why is this at top level?

Reply:
Go rules? I don't know why we have duplicate go/grpc rules throughout, seems to be an issue with go_rules.